src/main/kotlin/com/gaelmarhic/quadrant/QuadrantPlugin.kt

```diff
@@ -39,20 +39,20 @@ class QuadrantPlugin : Plugin<Project> {
         val variants = block(extension)
         val mainSourceSet = extension.sourceSet(MAIN_SOURCE_SET)
 
-        registerTask(createGenerateActivityClassNameConstantsTask(), variants)
+        gradle.projectsEvaluated {
+            registerTask(createGenerateActivityClassNameConstantsTask(), variants)
```

Comment:
Because of Gradle's lazy loading, we have to wait until all projects are evaluated to access the `namespace` property, not only for registering but also for creating the task.